lxd/images.go

```diff
@@ -2264,6 +2264,8 @@ func imageDelete(d *Daemon, r *http.Request) response.Response {
 		// Remove main image file from disk.
 		imageDeleteFromDisk(imgInfo.Fingerprint)
 
+		d.State().Events.SendLifecycle(projectName, lifecycle.ImageDeleted.Event(imgInfo.Fingerprint, projectName, op.Requestor(), nil))
```

Comment:
I believe we should also issue this event when an image is automatically deleted.

Reply:
Added it to `pruneExpiredImagesInProject`, and passed around the operation for the requestor.